karspexet/venue/management/commands/build_seats.py

```diff
@@ -0,0 +1,34 @@
+from django.core.management.base import BaseCommand
+from karspexet.venue.models import Venue, Seat, SeatingGroup
+
+import csv
```

Comment:
I'm not sure, but I always put standard libs first, then third party libraries and finally local stuff (also using `from .sibling import Foo` when I can). That and having pure `import module` above `from module import class`.